### ISSUE TITLE
Настроить авто апдейт

### ISSUE THREAD

**Issue by Big-Shark** [LaravelRUS/GitterBot#10]
Сделать так чтоб бота обновить на сервере мог любой человек из определенного списка людей

**Comment by SerafimArts** [comment]
@Big-Shark Мб добавить пулл + композер апдейт + рестарт бота (короче деплой) при пуше в мастер ветку, м?

**Comment by Big-Shark** [comment]
да, норм, гитзах может вызывать урл который все перезагрузит и обновит

**Comment by Big-Shark** [comment]
И допустим сам напишет об этом в чат для бота
